### PR TITLE
:art: Revamp Tutorials

### PR DESCRIPTION
**Pull request by alafage**
Tutorial Outline
==========

Featured on front page (6 slots available)
-------------------------

- [x] **Classification**
  - [x] Classification with Torch-Uncertainty (`tutorial_classification`)
  - [x] Deep Evidential Classification with Torch-Uncertainty (`tutorial_evidential_classification`)
  - [x] Classification with Bayesian Neural Networks (`tutorial_bayesian`)
  - [x] Out-of-Distribution Detection in Classification tasks (`tutorial_ood_detection`) - finish after merging #153 
  - [x] Image Classification with Packed-Ensembles (`tutorial_pe_cifar10`)
  - [x] Image Classification under Distribution Shift (`tutorial_distribution_shift`)
- [x] **Regression**
  - [x] Regression with Torch-Uncertainty (`tutorial_regression`)
  - [x] Deep Evidential Regression with Torch-Uncertainty (`tutorial_der_cubic`)
  - [x] Probabilistic Regression with Torch-Uncertainty (`tutorial_probabilistic_regression`)
- [x] **Segmentation**
  - [x] Semantic Segmentation with Torch-Uncertainty (`tutorial_muad_seg`)
  - [x] Semantic Segmentation with Deep Ensembles (`tutorial_muad_deep_en`)
  - [x] Semantic Segmentation with Packed-Ensembles (`tutorial_muad_packed`)
- [ ] **Pixel Regression**
  - [ ] Pixel Regression with Torch-Uncertainty
- [ ] **Ensemble Methods**
  - [ ] Benchmark Ensemble Methods on Classification (toy dataset)
  - [x] From Deep Ensembles to Packed Ensembles (`tutorial_from_de_to_pe`)
- [x] **Bayesian Methods**
  - [x] Train a Bayesian Neural Network in Three Minutes (`tutorial_bayesian`)
  - [x] Training a model with Monte Carlo Batch Normalization (`tutorial_mc_batch_norm`)
  - [x] Training a model with Monte Carlo Dropout (`tutorial_mc_dropout`)
  - [ ] Benchmark Bayesian Approaches on Classification (toy dataset)

Others
-------

- [ ] Data Augmentation
  - [x] Add Image Corruptions to your Dataset (`tutorial_corruptions`)
  - [ ] Test-time adaptation with Zero @fira7s
  - [ ] Mixing images and targets with Mixup
- [x] Model Calibration
  - [x] Improve Top-label Calibration with Temperature Scaling (`tutorial_scaler`)

### CODE REVIEW

File: auto_tutorial_source/Classification/tutorial_classification.py [hunk 1]
```diff
@@ -1,18 +1,9 @@
 """
-Training a LeNet with Monte-Carlo Dropout
-=========================================
+Training a LeNet for Image Classification with TorchUncertainty
+===============================================================
 
-In this tutorial, we will train a LeNet classifier on the MNIST dataset using Monte-Carlo Dropout (MC Dropout), a computationally efficient Bayesian approximation method. To estimate the predictive mean and uncertainty (variance), we perform multiple forward passes through the network with dropout layers enabled in ``train`` mode.
+In this tutorial, we will train a LeNet classfier on the MNIST dataset using TorchUncertainty.
```
Comment (o-laurent) [inline auto_tutorial_source/Classification/tutorial_classification.py:5]: classfier -> classifier

**Review by Seancoshea — APPROVED** [review]
